webpack.config.js

```diff
@@ -0,0 +1,76 @@
+const devCerts = require("office-addin-dev-certs");
+const { CleanWebpackPlugin } = require("clean-webpack-plugin");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const webpack = require("webpack");
+
+module.exports = async (env, options) => {
+    const dev = options.mode === "development";
```

Comment:
The official WebPack docs recommend having separate config files for development and production. Should we add a production config?

Reply:
I'm ok to not have it for now, since devs cannot publish their SSO add-ins yet. @Rick-Kirkham do you think this is a blocker?

Reply:
@nokafor We should minimize the differences between the basic Office-Addin-TaskPane and the addition of SSO. This discussion in not specific to SSO and it would be better to not make a one-off change for the SSO template only because this would make it seem that it is a requirement of SSO when it is not.

Reply:
I'm OK with just one webpack config for a preview API. But we should revisit this after the new year for all our templates.

Reply:
I am going to revert the recent webpack change I made to add 'const dev = options.mode === "development";' back in, so we are consistent with the webpack.config,js for all other project templates

Reply:
This must be the "dev variable" you guys were talking about in the other conversation. You never did say. ;-)

Reply:
Yes it is.